stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -13,20 +14,23 @@ class AssignManualQuants(models.TransientModel):
     @api.multi
     @api.constrains('quants_lines')
     def check_qty(self):
-        for record in self:
-            if record.quants_lines:
-                move = self.env['stock.move'].browse(
-                    self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
-                    raise exceptions.Warning(
-                        _('Quantity is higher than the needed one'))
+        precision_digits = dp.get_precision('Product Unit of Measure'
+                                            )(self.env.cr)[1]
```

Comment:
I am not sure of the usage of cursor here...

Reply:
Maybe something like this would be more standard:

```
precision_digits = self.env['decimal.precision'].precision_get('Product Unit of Measure')
```